releases/v1.3.0

```diff
@@ -0,0 +1,43 @@
+Highlights:
+* Improved Kotlin support (#319)
+* Context deadline exceeded gRPC error will be considered as non-retryable (#654) and will include details from the previous exception when possible (#674)
```

Comment:
Also worth mentioning in highlights that there is a backoff in the waiting for workflow completion in case of failures.